tests/publiccloud/ssh_interactive_start.pm

```diff
@@ -19,6 +19,7 @@ use version_utils;
 sub run {
     my ($self, $args) = @_;
     die "tunnel-console requires the TUNNELED=1 setting" unless (is_tunneled());
+    die "TEST FOR SAP CLEANUP";
```

Comment:
I assume this test line , but I will comment here anyway to increase chances that it will not get into master :)